lib/bundlex/project.ex

```diff
@@ -86,7 +83,12 @@ defmodule Bundlex.Project do
   defmacro __using__(_args) do
     quote do
       @behaviour unquote(__MODULE__)
+      @doc false
```

Comment:
Why have you put `@doc false` there?

Reply:
These functions are to be called by bundlex only. I changed the names to avoid potential conflicts